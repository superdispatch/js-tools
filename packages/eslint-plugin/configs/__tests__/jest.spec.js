const { testInheritance } = require('../__testutils__/test-eslint-config');

testInheritance('jest', 'base');
