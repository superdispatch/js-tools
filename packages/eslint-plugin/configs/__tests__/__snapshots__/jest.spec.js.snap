// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inheritance config 1`] = `
Snapshot Diff:
- First value
+ Second value

  Object {
-   "env": Object {},
+   "env": Object {
+     "jest": true,
+     "jest/globals": true,
+   },
    "globals": Object {},
    "parser": null,
    "parserOptions": Object {
      "ecmaVersion": 2018,
    },
    "plugins": Array [
      "@superdispatch",
      "prettier",
      "import",
+     "jest",
    ],
    "settings": Object {},
  }
`;

exports[`Inheritance dev rules 1`] = `
Snapshot Diff:
Compared values have no visual difference.
`;

exports[`Inheritance rules 1`] = `
Snapshot Diff:
- First value
+ Second value

@@ --- --- @@
    ],
+   "import/no-extraneous-dependencies": Array [
+     "error",
+     Object {
+       "devDependencies": true,
+       "peerDependencies": true,
+     },
+   ],
    "import/no-mutable-exports": Array [
@@ --- --- @@
      "off",
+   ],
+   "jest/no-alias-methods": Array [
+     "error",
+   ],
+   "jest/no-disabled-tests": Array [
+     "error",
+   ],
+   "jest/no-focused-tests": Array [
+     "error",
+   ],
+   "jest/no-identical-title": Array [
+     "error",
+   ],
+   "jest/no-jasmine-globals": Array [
+     "error",
+   ],
+   "jest/no-jest-import": Array [
+     "error",
+   ],
+   "jest/no-test-prefixes": Array [
+     "error",
+   ],
+   "jest/prefer-to-be-null": Array [
+     "error",
+   ],
+   "jest/prefer-to-be-undefined": Array [
+     "error",
+   ],
+   "jest/prefer-to-contain": Array [
+     "error",
+   ],
+   "jest/prefer-to-have-length": Array [
+     "error",
+   ],
+   "jest/valid-describe": Array [
+     "error",
+   ],
+   "jest/valid-expect": Array [
+     "error",
+   ],
+   "jest/valid-expect-in-promise": Array [
+     "error",
    ],
`;
